modules/ebi-metagenomics/combinedgenecaller/merge/main.nf

```diff
@@ -7,7 +7,7 @@ process COMBINEDGENECALLER_MERGE {
         'biocontainers/mgnify-pipelines-toolkit:0.2.0--pyhdfd78af_0' }"
 
     input:
-    tuple val(meta), path(prodigal_sco, stageAs: "pro.sco"), path(prodigal_ffn, stageAs: "pro.ffn"), path(prodigal_faa, stageAs: "pro.faa"), path(fgs_out, stageAs: "fgf.out"), path(fgs_ffn, stageAs: "fgf.ffn"), path(fgs_faa, stageAs: "fgs.faa"), path(mask_file)
+    tuple val(meta), path(prodigal_sco, stageAs: "prodigal/"), path(prodigal_ffn, stageAs: "prodigal/"), path(prodigal_faa, stageAs: "prodigal/"), path(fgs_out, stageAs: "fgsrs/"), path(fgs_ffn, stageAs: "fgsrs/"), path(fgs_faa, stageAs: "fgsrs/"), path(mask)
```

Comment:
is the `stageAs` just putting those files in respective subdirectories? e.g. for `prodigal_sco` it goes from `blabla.sco` to `prodigal/blabla.sco`? (making up names here haha)

Reply:
the problem is that is _very_ like that the files are called the same.. causing the process to fail.. os I place them in different folders to avoid that issue